src/garage/experiment/local_runner.py

```diff
@@ -321,7 +321,12 @@ def obtain_episodes(self,
                              ' or pass `batch_size` to runner.obtain_samples.')
         episodes = None
         if agent_update is None:
-            agent_update = self._algo.policy.get_param_values()
+            policy = getattr(self._algo, 'exploration_policy', None)
```

Comment:
is the reason we are able to do this because exploration_policy is a wrapper around a policy?

Reply:
yes

Reply:
Yeah, basically exploration policies are just policy wrappers, so to just works. I thought we already did this, but I guess I was wrong.